### ISSUE TITLE
Video chat is not working.

### ISSUE THREAD

**Issue by Abhishekkumar179** [ShubhamCpp/Real-Time-Chat-WebRTC#1]
Hi Shubham,
I have tried your code in my pc but only chat part is working and video part is not working please help me how can i use video chat also.

**Comment by abbeyseto** [comment]
Yeah, same with me too. Only chat works and if you type another room name on the second browser, it still connects to the former room
